src/lib/metrics.ts

```diff
@@ -807,6 +812,9 @@ export function registerPrometheusMetrics(
         clientDeltaMemory.reset();
         clientDeltaMemory.set(event.memory);
     });
+    eventBus.on(events.CLIENT_REGISTERED, ({ appName, environment }) => {
```

Comment:
I'm assuming we don't need to batch those before calling inc

Reply:
no need, its just in-memory counters.